src/base-element.js

```diff
@@ -513,6 +527,24 @@ export class BaseElement {
     this.element.setReadyStateInternal(state, opt_failure);
   }
 
+  /**
+   * Load heavy elements, perform expensive operations, add global
+   * listeners/observers, etc.
```

Comment:
Please add a comment that this is meant to be called multiple times, eg `mount` should handle the things `unmount` does.